### PR TITLE
Update hotTakeData.json

### PR DESCRIPTION
**Pull request by Kuraky**

### CODE REVIEW

File: hotTakeData.json [hunk 1]
```diff
@@ -100,7 +110,9 @@
     "LESS",
     "Angular",
     "React",
-    "Vue"
+    "Vue",
+    "Malbolge",
+    "Zaskroniec"
```
Comment (lucypoulton) [inline hotTakeData.json:115]: is this a language? google is giving me results for snakes

File: hotTakeData.json [hunk 2]
```diff
@@ -253,6 +277,14 @@
...
     "Why are {language} developers paid so much even though they don't understand {thing}?",
-    "The {problem} issue is just a thinly veiled advertisement for {thing}."
+    "The {problem} issue is just a thinly veiled advertisement for {thing}.",
+    "{person} debunked {technology} a long time ago.",
+    "{person} was the first true computer scientist.",
+    "{thing} fans are quickly taking over the tech world and that's good.",
+    "{thing} fans are quickly taking over the tech world and that's bad.",
+    "{technology} users couldn't even print Hello World in {language}.",
+    "0xDEADBEEF > {thing}",
+    "{problem} this, {problem} that. How about you learn {thing} instead?",
```
Comment (lucypoulton) [inline hotTakeData.json:287]: I don't think this will capitalise, maybe worth putting something at the front?

File: hotTakeData.json (hunk 1 above)
Comment (Kuraky) [reply hotTakeData.json:115]: https://github.com/kspalaiologos/Zaskroniec

File: hotTakeData.json (hunk 2 above)
Comment (Kuraky) [reply hotTakeData.json:287]: true, gonna think this through

Comment (knightzmc) [reply hotTakeData.json:287]: It does capitalise automatically

**Review by knightzmc — APPROVED** [review]

**Pull request merged**